examples/contour/01-contour-config.yaml

```diff
@@ -120,6 +120,12 @@ data:
     #   service fails to respond with a valid rate limit decision within
     #   the timeout defined on the extension service.
     #   failOpen: false
+    # Defines whether to include the X-RateLimit headers X-RateLimit-Limit,
```

Comment:
Just one silly nit to align the help text with the variable like the other options.

Reply:
fixed!